scss/_patterns_chip.scss

```
@@ -10,6 +10,7 @@
     align-items: baseline;
     border-radius: 1rem;
     display: inline-flex;
+    gap: $spv--medium;
```

```suggestion
    gap: $spv--small;
```

variables starting with `spv` are for vertical spacing, h for horizontal. In addition, I think .75 is a bit much so the suggestion above reduces it to .5rem;

---

done!